test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java

```diff
@@ -409,6 +419,100 @@ public void narrativeDecoration() throws Exception {
         NarrativeInfoDecoratorTest.compare(res.getAccessGroupNarrativeInfo(), expected);
     }
 
+    @Test
+    public void optionalAuthForSearch() throws Exception {
```

Comment:
This method tests with noAuthSearchCli. I think we need one more method that tests with searchCli and an authenticated user. Perhaps call the methods unauthenticatedSearchTest() and authenticatedSearchTest()

Reply:
Have added more tests to get and verify search results with authentication.